source/includes/documentation/_vue_js_elements.md

```diff
@@ -2964,3 +2964,89 @@ Available methods in Java JDI Light:
 
 For examples of usage see: [JDI Vuetify Chip tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/ChipsTests.java).
 
+### 5.40 Badge
+
+[Badge Vuetify documentation page](https://vuetifyjs.com/en/components/badges/)
+
+Badge is located in the following class:
+
+- __Java__: _com.epam.jdi.light.vuetify.elements.common.Badge.java_
+
+```java
+    @Test
+    public void typeBadgesTest() {
+        lockUnlockAccountBadge.is().icon();
+        dotBadge.has().notIcon();
+        imageBadge.has().avatar();
+        dotBadge.has().notAvatar();
+        dotBadge.is().dot();
+        imageBadge.is().notDot();
+    }
+
+    @Test
+    public void textBadgesTest() {
+        Badge simpleBadge = simpleBadges.get(1);
+        simpleBadge.show();
+        simpleBadge.has().text("1");
+    }
+```
+
+__Badges__ - The `v-badge` component superscripts or subscripts an avatar-like icon or text onto content to highlight information to a user or to just draw attention to a specific element.
+Content within the badge usually contains numbers or icons.
+
+![Badges example](../../images/vuetify/badges.png)
+
+__Vuetify v2.6.14__ code example:
+```html
+<template>
+  <v-container>
+    <v-row
+      align="center"
+      justify="center"
+    >
+      <v-badge
+        bordered
+        color="error"
+        icon="mdi-lock"
+        overlap
+      >
+        <v-btn
+          class="white--text"
+          color="error"
+          depressed
+        >
+          Lock Account
+        </v-btn>
+      </v-badge>
+    </v-row>
+  </v-container>
+</template>
```

Comment:
Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 

![image](https://user-images.githubusercontent.com/55189259/212974214-d3398fb4-8119-4be5-a554-b2eea1d9effd.png)